dbus/mako.c

```diff
@@ -468,10 +470,7 @@ void emit_modes_changed(struct mako_state *state) {
 
 static const sd_bus_vtable service_vtable[] = {
 	SD_BUS_VTABLE_START(0),
-	SD_BUS_METHOD("DismissAllNotifications", "", "", handle_dismiss_all_notifications, SD_BUS_VTABLE_UNPRIVILEGED),
-	SD_BUS_METHOD("DismissGroupNotifications", "", "", handle_dismiss_group_notifications, SD_BUS_VTABLE_UNPRIVILEGED),
-	SD_BUS_METHOD("DismissLastNotification", "", "", handle_dismiss_last_notification, SD_BUS_VTABLE_UNPRIVILEGED),
-	SD_BUS_METHOD("DismissNotification", "ub", "", handle_dismiss_notification, SD_BUS_VTABLE_UNPRIVILEGED),
+	SD_BUS_METHOD("Dismiss", "a{sv}", "", handle_dismiss, SD_BUS_VTABLE_UNPRIVILEGED),
```

Comment:
Nit: I'd prefer to keep "notification" in the name. Perhaps "DismissNotifications" since it might dismiss one or more?

Reply:
I changed it to `DismissNotifications`